src/main/java/com/exasol/adapter/sql/SqlColumn.java

```diff
@@ -7,16 +7,27 @@
     private final int id;
     private final ColumnMetadata metadata;
     private String tableName;
+    private String tableAlias;
 
     public SqlColumn(final int id, final ColumnMetadata metadata) {
         this.id = id;
         this.metadata = metadata;
+        this.tableName = null;
+        this.tableAlias = null;
```

Comment:
DRY: Have one main constructor (the one with the most parameters), make all others call that one.